src/types/runtime/index.ts

```diff
@@ -56,7 +56,7 @@ export interface RuntimeControls {
   setCellProperty(opts: SetCellPropertyOptions): boolean;
   resetCell(opts: ResetCellOptions): boolean;
   runCell(opts: RunCellOptions): boolean;
-  focusCell(opts: FocusCellOptions): boolean;
+  focusCell(opts: FocusCellOptions): Promise<boolean>;
```

Comment:
Not 100% sure about this change.. I understand the need for it - I guess it's relevant when you are modifying the cells and then want to focus one. Maybe this should be the caller's responsibility?

Reply:
I'm also not entirely sure. I suppose putting a warning about waiting for the notebook to be done updating into the documentation comment would also work.